dpe/src/printer.rs

```diff
@@ -0,0 +1,33 @@
+// Licensed under the Apache-2.0 license
+
+use ufmt::uWrite;
+
+use platform::{Platform, PlatformError};
+
+use crate::dpe_instance::DpeTypes;
+
+pub struct Printer<'a, T: DpeTypes + 'a> {
+    pub platform: T::Platform<'a>,
+}
+
+impl<'a, T: DpeTypes> uWrite for Printer<'a, T> {
+    type Error = PlatformError;
+
+    fn write_str(&mut self, str: &str) -> Result<(), Self::Error> {
+        self.platform.write_str(str)?;
+        Ok(())
```

Comment:
This can be simplified

```suggestion
        self.platform.write_str(str)
```

Reply:
Done.